lib/core/src/input/input.tsx

```diff
@@ -123,7 +123,7 @@ const inputRender = (
 ): JSX.Element => {
 	validate(props);
 	const size = props.size ?? Input.sizes.medium;
-	const rawProps = omit(props, ["size", "style"]);
+	const rawProps = omit(props, ["size", "style", "setValue", "icon", "list"]);
```

Comment:
Can you also exclude "onChange" and "value" too? We already override them below, but I think it's better to explicitly exclude them here

Reply:
yeah, done 🥳